src/main/java/dev/thomazz/pledge/network/PacketFrameOutboundTailHandler.java

```diff
@@ -81,11 +81,9 @@ public void drain(ChannelHandlerContext ctx, @Nullable PacketFrame frame) throws
         }
 
         // Drain all queued packets after the tail handler
-        ChannelHandlerContext target = ctx.pipeline().context(PacketFrameOutboundTailHandler.HANDLER_NAME);
```

Comment:
You want to flush these packets from the tail handler context, otherwise these packets will go through the entire pipeline again

Reply:
Ok, fixed! I just thought it was the same thing as ctx, since they were both getting the tail handler.

Reply:
You're right actually, the context passed in is already from the tail handler. This is a remnant of some old code confusing me...